clang/test/Driver/sycl-offload-intelfpga.cpp

```diff
@@ -73,7 +75,7 @@
 // CHK-FPGA: clang-offload-bundler{{.*}} "-type=o" "-targets=host-x86_64-unknown-linux-gnu,sycl-spir64_fpga-unknown-unknown-sycldevice" {{.*}} "-outputs=[[FINALLINK2:.+\.o]],[[OUTPUT1:.+\.o]]" "-unbundle"
 // CHK-FPGA: llvm-no-spir-kernel{{.*}} "[[OUTPUT1]]" "-o" "[[OUTPUT3:.+\.o]]"
 // CHK-FPGA: llvm-link{{.*}} "[[OUTPUT3]]" "-o" "[[OUTPUT4:.+\.bc]]"
-// CHK-FPGA: llvm-spirv{{.*}} "-o" "[[OUTPUT5:.+\.spv]]" "[[OUTPUT4]]"
+// CHK-FPGA: llvm-spirv{{.*}} "-o" "[[OUTPUT5:.+\.spv]]" {{.*}} "[[OUTPUT4]]"
```

Comment:
I swapped arguments in SPIRV translation job, but since I'm doing to rewrite function which constructs llvm-foreach job, I think it can be returned back.

Reply:
Actually, It happened because I moved out spirv translation action from device linking action. It means that there is another function which constructs spirv translator action now, and it makes it in other way.

Reply:
Looks like lines 7148-7149 from Clang.cpp, function `SPIRVTranslator::ConstructJob` are responsible. I'd suggest the following to comply with those:
```suggestion
// CHK-FPGA: llvm-spirv{{.*}} "-o" "[[OUTPUT5:.+\.spv]]" "-spirv-max-version=1.1" "-spirv-ext=+all" "[[OUTPUT4]]"
```

Reply:
Okay, added.